tensorflow/python/keras/backend.py

```diff
@@ -65,6 +65,7 @@
 
 py_all = all
 py_sum = sum
+py_any = any
```

Comment:
Can you modify the PR description to briefly describe what bug this change is fixing?

Reply:
This is standard practice, as you can see in the lines above. `all`, `sum` and `any` backend functions overrides python's built in `all`, `sum` and `any`.

Reply:
No, I was talking about this pull request in general. What bug is it trying to fix? If there is a GitHub issue, please link to it.

Reply:
You can see that the tests (added in this PR) will fail without the changes to backend.py. These changes have already been made to keras-team/keras.

Reply:
I'm aware of the current bugs in Dot. I just want to make sure the commit message documents the change better.